src/Stripe.net/Infrastructure/RuntimeInformation.cs

```diff
@@ -110,27 +112,7 @@ internal static string GetFullFrameworkVersion()
 
         internal static string MapToReleaseVersion(string servicingVersion)
         {
-            // the following code assumes that .NET 4.5 is the oldest supported version
-            if (string.Compare(servicingVersion, "4.5.1") < 0)
-            {
-                return "4.5";
-            }
-
-            if (string.Compare(servicingVersion, "4.5.2") < 0)
-            {
-                return "4.5.1";
-            }
-
-            if (string.Compare(servicingVersion, "4.6") < 0)
-            {
-                return "4.5.2";
-            }
-
-            if (string.Compare(servicingVersion, "4.6.1") < 0)
-            {
-                return "4.6";
-            }
-
+            // the following code assumes that .NET 4.6.1 is the oldest supported version
```

Comment:
Unrelated, but since 4.6.1 is the minimum required version now we can remove a bunch of code.